index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Registration Form</title>
+    <link rel="stylesheet" href="css/style.css">
+    
+  </head>
+
+  <body>
+    <header>
+      <h1 id="title">Microverse Registration Form</h1>
+      <p id="description">Please register your information</p>
+    </header>
+      <form id="suervey-form">          
```

Comment:
Children elements should be aligned after the parent element.

Reply:
Hi @ommarz 👋,

Kindly follow the suggestion, asked by the previous reviewer.
- [ ] Please use `kind words`, `be clear`, and always `give a clear explanation of why the requested change is necessary`.

For example, you could say: ⬇️

- Good job so far, 👏 However, Please ensure that you align your child element properly, child elements are typically placed within the parent element and rendered in the order they appear in the HTML markup. By default, child elements will flow one after another vertically within the parent element, unless you specify a different layout. Please resolve it. 🙂